src/website/app/pages/ComponentStatus/componentStatus.md

```diff
@@ -23,20 +23,21 @@ Statuses:
 
 | Component | Status |
 |---|---|
+| NumericInput | [planned] |
```

Comment:
IMHO, available items should come before planned.  It flows better logically - **done** -> **doing** -> **will do** -> **might do**

Reply:
That's how I had it originally, but @Mikewaite pointed out that people coming to the page probably want to see what we're working on since the "completed" components are displayed in the left nav already. Which also means that once something is "in development", it'll be at the top.

Reply:
I disagree with @Mikewaite  - I think it should go in regular order, otherwise it feels like arbitrary order.

Reply:
That's fine. I can roll over on this. 

Done > Doing > Will Do > Might do